### DISCUSSION TITLE
I can only access a limited number of users on client workstations

### DISCUSSION THREAD

**Discussion by Prof102** [ltsp/ltsp#879] [General]
Is the number of users limited?

Hello, 

I created the users using the "createguests.sh" script.
I need 25 users, so I modified the following command:
"for ip in {20..250}; do" to "for ip in {1..25}; do"

On my server, I can access all 25 users, but on the clients I only have 20 users available. If I manually enter (eg) "user21" on a client, I get an error message: "user unknown".

My server is a Dell Precision with 16 G° of RAM; I'm running Debian12 on a .
250 G° SSD. 

I must have made an error, but I can't find it. Can you help me to solve that ?

Thanks

**Comment by alkisg** [comment]
After adding users on the server, run `sudo ltsp initrd` and reboot the clients.

**Reply by Prof102** [reply]
Thank you, everything ok now; works like a charm !!
